### PR TITLE
Purple tutorial theme

### PR DESCRIPTION
**Pull request by jwunderl**
As discussed in design office hours, the orange is jarring & is very inaccessible / conflicts with other editor colors. This switches the tutorial colors to be the microbit purple colors as we had talked about there

* fix https://github.com/microsoft/pxt-microbit/issues/3949
* fix https://github.com/microsoft/pxt-microbit/issues/2290
* fix color bunch of colors: tutorial preview in my projects gallery, detail view, tutorial buttons, etc (no issue filed but is currently inaccessible)

build: https://makecode.microbit.org/app/8674fc02997b6643600aa3d7182a320ea5887d60-f82ef95661

image
image
image
image

### CODE REVIEW

**Comment by shakao** [comment]
hmm i think the js/python "start tutorial" cards should be a different color still... or maybe "blocks" should be green? i think it's useful to differentiate

**Comment by jwunderl** [comment]
Ah yeah I had it as a darker purple at first but dropped it cause it looked bad, green could work. For what it's worth arcade is all orange currently / could be fixed to match as well

File: theme/style.less [hunk 1]
```diff
@@ -57,13 +57,31 @@
...
+    .detailview .actions .card-action .button.approve {
+        background-color: @mainMenuTutorialBackground;
+    }
+    .tutorial-progress.orange {
+        background-color: @mainMenuTutorialBackground !important;
+        border-color: @mainMenuTutorialBackground !important;
+    }
+}
+
+.hc #homescreen .ui.home .tutorial-progress.orange {
```
Comment (shakao) [inline theme/style.less:74]: small: slight preference for keeping hc styles in the highcontrast.less file

**Comment by shakao** [comment]
oh yeah! didn't realize arcade was all the same color--maybe it's fine then!

File: theme/style.less (hunk 1 above)
Comment (jwunderl) [reply theme/style.less:74]: Yeah, I had it in here as it's microbit specific override but I'll bump it over there as it seems possible to change in other targets later on too. I'll add it on to https://github.com/microsoft/pxt/pull/8085 

also I checked green first button and I think I like it, eye catching & nice parallel with completed / started tutorials. Adding that in 
image

Comment (shakao) [reply theme/style.less]: lsdkfjf ok now looking at this, i'm vaguely remembering that it used to be green and we recolored it (https://github.com/microsoft/pxt/pull/6908) BUT that was pre-having-a-real-designer so maybe this is worth a quick pass in design office hrs..... ?

**Comment by Jaqster** [comment]
I do think it's strange that Blocks tutorials are a different color...  I would keep all the tutorials the same color.  From a user perspective, there are 3 options - why is one a different color?

File: theme/style.less (hunk 1 above)
Comment (jwunderl) [reply theme/style.less]: Ah sounds good, I dropped the style from this pr for now / will probably want to walk through all these changes with Sonia real quick before merging anyway just to confirm~

**Comment by jwunderl** [comment]
Feedback from design meeting was just changing the color for the step indicator for contrast / prettiness:

image

added in last commit ~

**Comment by abchatra** [comment]
@jwunderl ready to merge?

**Review by abchatra — APPROVED** [review]

**Comment by abchatra** [comment]
Let me merge it for now. We can test and decide tomorrow.

**Pull request merged**